src/site/markdown/how-to.md

```diff
@@ -70,8 +70,7 @@ Add all the project specific source code files in `<glob>`. Scala in this case:
 
 ### Adding source directory for bespoke project layouts
 
-In most of the cases incremental Maven will recognize directories automatically by build introspection. If not, you can
-add additional directories with `<include>`. Also you can filter out undesirable dirs and files by using exclude tag
+In most cases, incremental Maven automatically recognizes directories by introspecting the build. When it is not enough, adding additional directories with `<include>` is possible. Also, you can filter out undesirable dirs and files by using exclude tag.
```

Comment:
`incremental Maven` -> `the build-cache extension`

Reply:
fixed